pkg/distributor/distributor.go

```diff
@@ -79,6 +79,12 @@ var (
 	})
 )
 
+type clusterTracker interface {
+	LookupInstance(ctx context.Context, cluster string, instance string) *instanceTracker
+	setTimeout(overwrite, write int64)
```

Comment:
This seems to never be called?

Reply:
I'm using it in `ha_tracker_test.go`. I can remove it and just modify the values on the struct in line, or remove the function from the interface and just have it local within the test file.

Reply:
> or remove the function from the interface and just have it local within the test file.

Yeah, this sounds better.